docs-source/content/userguide/managing-operators/installation/_index.md

```diff
@@ -86,6 +86,30 @@ $ helm install weblogic-operator weblogic-operator/weblogic-operator
 
 #### Upgrade the operator
 
+{{% notice note %}}
+Because operator 3.0.0 introduces _non-backward compatible_ changes, you can not use `helm upgrade` to upgrade
+an operator using 2.6.0 to an operator using 3.0.0. Instead, you must delete the 2.6.0 operator and then install the
```

Comment:
(Again, here I would suggest the less verbose way of describing these operator versions)  -> a 2.6.0 operator to a 3.0.0 operator